### DISCUSSION TITLE
moes ms-105z dimmer switch module

### DISCUSSION THREAD

**Discussion by Mikescotland** [Koenkk/zigbee2mqtt#14522] [Devices]
Hello.
Is it possible to integrate to this device also original settings? 
Bulb type and minimum brightness is missing. Here is json from tuya:

bright_value_1 | Integer | {   "min": 10,   "max": 1000,   "scale": 0,   "step": 1 }
brightness_min_1 | Integer | {   "min": 10,   "max": 1000,   "scale": 0,   "step": 1 }
led_type_1 | Enum | {   "range": [     "led",     "incandescent",     "halogen"   ] }
countdown_1 | Integer | {   "unit": "s",   "min": 0,   "max": 86400,   "scale": 0,   "step": 1

**Comment by Koenkk** [comment]
This has been added in the latest dev branch already. (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)
